### ISSUE TITLE
DigiKam won't read or write PNG or GIF tags

### ISSUE THREAD

**Issue by acloudadmin** [Dijji/FileMeta#114]
Hi there,

Great work on this -- incredibly useful. Thank you.

I'm wondering, though, if you have any thoughts on why the photo library program DigiKam will neither read nor write tags to or from gifs or pngs.

Using filemeta, I used the 'simple' setting to add the handlers to both PNGs and GIFs. In Windows 10, I can now write tags to both file formats. But digiKam fails to pick up the tags I've set in Windows explorer. Neither will it write the tags I've assigned to those files in digiKam to the files themselves.

This might actually be a question for the DigiKam developers, but thought I'd check with you first.

Thanks again!

-Andy

**Comment by Dijji** [comment]
In general, third-party applications do pick up tags and other properties set using File Meta, because they use the Windows property system to access them.

However, I can understand why DigiKam might not do so. Because it can manipulate photographs, it knows how to read the file formats, and therefore probably does not look for any data from outside the file itself. And since gif and png files do not, as far as I know, store tags as part of their native format, DigiKam probably does not expect to read them for a photograph. When it comes to setting tags, I notice that DigiKam works by creating its own database, cataloguing the photographs, and I strongly suspect that that is where it keeps the tag values that you set.

To sum up, my suspicion is that DigiKam ignores the Windows property system as far as tags are concerned, and therefore does not interact with File Meta tags. Since DigiKam are open source, it would be interesting to know what their developers feel about the issue.
